docs/howto/features/buckets.md

```diff
@@ -276,16 +276,26 @@ other cloud providers when needed.
 
 ### AWS
 
-On AWS, we would need to set up [cross account S3 access](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/).
+On AWS, we would need to set up [cross account S3 access](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/). It requires action from both the 2i2c engineering team as
+well as the the community representative.
 
-1. Find the ARN of the service account used by the *users* on the hub. You can
-   find this under `userServiceAccount.annotations.eks.amazon.com/role-arn` in
-   the `values.yaml` file for your hub. It should look something like
+1. **Who**: 2i2c Engineering Team
+
+   Find the ARN of the service account that is assumed by the users on the hub,
+   under `userServiceAccount.annotations.eks.amazon.com/role-arn` in the YAML
+   config specific to the hub. It should look something like
    `arn:aws:iam::<account-id>:role/<hub-name>`.
-2. In the AWS account *with the S3 bucket*, [create an IAM
-   policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html)
-   that grants appropriate access to the S3 bucket from the hub. For example, the
-   following policy grants readonly access to the bucket for users of the hub
+
+   If this hub has an equivalent *staging* hub, find the ARN of the service
+   account for the staging hub too. This helps us keep staging and prod in sync.
+
+2. **Who**: 2i2c Engineering Team
+
+   Generate a [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html)
+   that can be communicated to the community champion. You need to generate one policy
+   for each ARN determined in step 1. The `Action` section lists various permissions to be
+   granted. The default is to allow hub users to list all objects and get data - so readonly
+   access. The community champion is welcome to modify this to fit their needs.
```

Comment:
```suggestion
   for each ARN determined in step 1.
```

The description of action ocurring twice (but not principal/resource) - I suggest removing a mention of what action does here and relying on the description below instead saying the same thing.